app/policies/article_policy.rb

```diff
@@ -34,6 +34,10 @@ def permitted_attributes
        archived]
   end
 
+  def subscriptions?
+    user_is_author? || user_admin?
```

Comment:
This can easily be extended to allow organization admins to see subscriptions for articles they didn't write themselves in the future.